ec2-metadata

```diff
@@ -198,7 +199,7 @@ while [ "$1" != "" ]; do
 																 ;;
 	-g | --tags )                  print_tags
 																 ;;
-	-h | --help )                  print_help
+	     --help )                  print_help
```

Comment:
I don't see a reason for removing `-h` functionality here for help.

Reply:
Because `-h` is already used for `--local-hostname`